File-Formats/Presentation/Working-with-sections.md

```diff
@@ -423,104 +312,72 @@ You can move the sections within a PowerPoint presentation. The following code e
 {% tabs %}
 
 {% highlight c# tabtitle="C#" %}
-
 //Loads a PowerPoint presentation
 IPresentation pptxDoc = Presentation.Open("PPTXWithSections.PPTX");
-
 //Moves the second section to third position within the PowerPoint presentation.
 pptxDoc.Sections[2].Move(3);
-
 //Saves the PowerPoint presentation
 pptxDoc.Save("Sections.PPTX");
-
 {% endhighlight %}
 
 {% highlight vb.net tabtitle="VB.NET" %}
-
 'Loads a PowerPoint presentation
-Dim pptxDoc As IPresentation = Presentation.Open("PPTXWithSections.PPTX")
-
-'Moves the second section to third position within the PowerPoint presentation.
-pptxDoc.Sections(2).Move(3)
-
-'Saves the PowerPoint presentation
-pptxDoc.Save("Sections.PPTX")
-
+Dim pptxDoc As IPresentatio
```

Comment:
You have removed this VB code snippet. Please add it.

Reply:
@Suriya-Balamurugan -Done